cmd/podman/main_cgroups_unsupported.go

```diff
@@ -0,0 +1,15 @@
+//go:build windows || darwin || freebsd
+
+package main
+
+import (
+	"github.com/sirupsen/logrus"
+	"go.podman.io/common/pkg/cgroups"
+)
+
+func checkSupportedCgroups() {
+	unified, _ := cgroups.IsCgroup2UnifiedMode()
+	if !unified {
+		logrus.Debugln("Non-linux environment. Non-fatal cgroups check")
```

Comment:
This will _always_ trigger with the current implementation… “Non-fatal cgroups check” is not very useful to readers not familiar with this implementation.

Reply:
updated.